recipe/meta.yaml

```
@@ -1,44 +1,42 @@
 {% set name = "PasteDeploy" %}
-{% set version = "2.0.1" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d423fb9d51fdcf853aa4ff43ac7ec469b643ea19590f67488122d6d0d772350a
+  sha256: 9ddbaf152f8095438a9fe81f82c78a6714b92ae8e066bed418b6a7ff6a095a95
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: true
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
-    - setuptools
-    - pytest-runner
+    - setuptools >=41
   run:
     - python
```

```suggestion
    - python
    - importlib-metadata  # [py<38]
```

The selector `# [py<38]` is needed if the dependency required only for specific Python versions. Even if we don't support Python 3.7 and 3.8 we should use the dependency following the upstream source (what the maintainers say [here](https://github.com/Pylons/pastedeploy/blob/3.1/setup.cfg#L49-L50C5))

---

Added